ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+
+import React from 'react';
+import ReactTable from 'react-table';
+
+const GraphAlternateView = ({ graphData, colNum }) => {
+  const columnsOne = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count per Push',
+      accessor: 'y',
+    },
+  ];
+
+  const columnsTwo = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count',
+      accessor: 'y',
+    },
+    {
+      Header: 'Push Count',
+      accessor: 'z',
+    },
+  ];
```

Comment:
Please use better names than `x`, `y`, and `z`.  I suggest `date`, `failureCount` and `pushCount`.